docs/source/developer/render.rst

```diff
@@ -1,20 +1,59 @@
 Render Service
 ##############################################################################
 
+Production Infrastructure
+******************************************************************************
```

Comment:
Please use the same heading level symbols as the rest of the documentation, see `authors/topics.rst` for lots of levels.